config/settings.yml

```diff
@@ -358,6 +358,7 @@ claims_api:
 redis:
   host: localhost
   port: 6379
```

Comment:
Any objection to changes this to a single URL 

```
redis:
  url: localhost:6379
```

and the corresponding changes in deployment files? It can be a lot easier to work with multiple Redis instances in development particularly if you can use the /0../15 syntax to access additional databases.

That would let you do something like

```
redis:
  url: localhost:6379

redis_secondary
  url: localhost:6379/1

redis_sidekiq:
  url: localhost:6379/2

etc
```

Means you could also just cut/paste Elasticache URLs directly instead of having to break it into URL and port :)

Reply:
FWIW I'm already using `url` in the PR for separating Redis out but thought updating here made sense to keep them a little more in sync. Happy to help w/ devops PRs if that's a help

Reply:
Seems 👍 to me. My only preference would be that we do this work in a separate PR to minimize the moving parts.

Reply:
Whipped up a draft PR here: https://github.com/department-of-veterans-affairs/vets-api/pull/4219

Reply:
☝️ PR merged 👍